### ISSUE TITLE
Managing the first major release in CI

### ISSUE THREAD

**Issue by taorepoara** [convco/convco#290]
**Is your feature request related to a problem? Please describe.**

I use convco in my CI to automate the release management.

I do not use preMajor version (`0.1.0` for example) so my first release should be a `1.0.0`.

For now, there is no way to create a `1.0.0` without using the `--major` flag, but I can't use it since it's used in a CI.

It would be nice to have a way to create a `1.0.0` without changing the CI management.

**Describe the solution you'd like**

IMO, the simplest way to manage that is to prevent managing pre-major versions when the `preMajor` configuration is set to false in the config file (instead off forcing it based on the previous release).

The problem of this solution is that it wouldn't work to create the `1.0.0` on projects that uses pre-major versions.

**Describe alternatives you've considered**

Adapting the CI to add the possibility to set the `--major` flag based on a env var (that would work on Gitlab, but not on Github).

**Additional context**

**Comment by hdevalke** [comment]
What about adding an config option or a cli option `--initial-version` or an env var `CONVCO_INITIAL_VERSION` which all default to `0.1.0`?

**Comment by hdevalke** [comment]
To add this feature we just have to replace the version with the one specified here: https://github.com/convco/convco/blob/f40634653d28899604ab0c0baea47a61ad083071/src/cmd/version.rs#L199

**Comment by taorepoara** [comment]
@hdevalke 

Yes, creating a `CONVCO_INITIAL_VERSION` would be a good solution.

What is the `preMajor` config made for ?

**Comment by hdevalke** [comment]
I used the config spec from https://github.com/conventional-changelog/conventional-changelog-config-spec/blob/master/versions/2.1.0/README.md#premajor-boolean it seems indeed unused here

**Comment by taorepoara** [comment]
> I used the config spec from https://github.com/conventional-changelog/conventional-changelog-config-spec/blob/master/versions/2.1.0/README.md#premajor-boolean it seems indeed unused here

Can't we use it in convco conf to define if the project uses pre-major versions ?

**Comment by hdevalke** [comment]
it has a different use case. It is set for each commit to render the template for the changelog.
I'll add `CONVCO_INITIAL_VERSION` as env var `--initial-version` as cli flag and `initialVersion` in config.

**Comment by taorepoara** [comment]
Works fine, thanks !
